apis/cache/v1beta1/cloudmemorystore_instance_types.go

```diff
@@ -175,9 +175,9 @@ type CloudMemorystoreInstanceStatus struct {
 // A CloudMemorystoreInstance is a managed resource that represents a Google
 // Cloud Memorystore instance.
 // +kubebuilder:subresource:status
-// +kubebuilder:printcolumn:name="STATUS",type="string",JSONPath=".status.state"
```

Comment:
I think we can use `// +kubebuilder:printcolumn:name="STATUS",type="string",JSONPath=".status.atProvider.state"`

Reply:
good call!